### PR TITLE
MGDOBR-527 - Report Portal integration for Cucumber tests

### PR DESCRIPTION
**Pull request by jiripetrlik**
[MGDOBR-527](https://issues.redhat.com/browse/MGDOBR-527)

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [*] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [*] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [*] All new functionality is tested
- [*] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [*] Pull Request contains link to the JIRA issue
- [*] Pull Request contains link to any dependent or related Pull Request
- [*] Pull Request contains description of the issue
- [*] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [*] Check if code and Github action workflows have been modified and if the modification is safe
- [*] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.

</details>

### CODE REVIEW

File: integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/RunCucumberTest.java [hunk 1]
```diff
@@ -11,5 +11,6 @@
 @IncludeEngines("cucumber")
 @SelectClasspathResource("features")
 @ConfigurationParameter(key = PLUGIN_PROPERTY_NAME, value = "pretty")
+@ConfigurationParameter(key = PLUGIN_PROPERTY_NAME, value = "com.epam.reportportal.cucumber.ScenarioReporter")
```
Comment (sutaakar) [inline integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/RunCucumberTest.java:14]: It looks like the previous `pretty` logger is ignored, see the PR check log.
Can you try to use `@ConfigurationParameter(key = PLUGIN_PROPERTY_NAME, value = "pretty,com.epam.reportportal.cucumber.ScenarioReporter")` and check that it works?

Comment (jiripetrlik) [reply integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/RunCucumberTest.java]: @sutaakar 
Thank you for your tip. I've updated the PR and both seems working.

**Review by sutaakar — APPROVED** [review]

**Review by jomarko — APPROVED** [review]
Thank you for a PR @jiripetrlik

**Review by danielezonca — APPROVED** [review]
I don't know Report Portal but the PR looks safe to me 👍

**Pull request merged**